Assets/Scripts/EnemyBehaviour/EnemyBehaviour.cs

```diff
@@ -4,10 +4,11 @@
 
 public class EnemyBehaviour : EntityBehaviour
 {
+    public int maxHealthIncreasePerLoop = 2;
```

Comment:
this could probably be a constant?

Reply:
This is an adjustable value to vary between enemies and bosses (as enemyBehaviour is a base class of BossBehaviour).